definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-prod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "31/08/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "PreviousIssueDate",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
```

Comment:
what about "D8Cohort"?

Reply:
Previously there were no permissions for d8Cohort for the solicitor

Reply:
ok